imports/plugins/core/versions/server/migrations/index.js

```diff
@@ -16,3 +16,4 @@ import "./15_update_shipping_status_to_workflow";
 import "./16_update_billing_paymentMethod";
 import "./17_set_shop_uols";
 import "./18_use_react_for_header_and_footer_layout";
+import "./20_set_shop_uows";
```

Comment:
doesn't really matter, but where is migration 19?

Reply:
I believe that was in a PR that ended up being closed/not merged